.circleci/config.yml

```diff
@@ -4,7 +4,7 @@ version: 2.1
 setup: true
 
 orbs:
-    gravitee: gravitee-io/gravitee@2.1
+    gravitee: gravitee-io/gravitee@4.0.0
```

Comment:
I think the 4.1.0 is to be used for plugins now @gaetanmaisse do you concur

Reply:
Yes!